bot.py

```diff
@@ -102,13 +103,19 @@ def run():
                 val, coords = 0, None
 
             # Update the maximum value and corresponding image file and coordinates if necessary
-            if val > max_val:
+            # if val > max_val:
+            if ('max_number_of_games_played_text.en.png' == image_file and val > 0.42):
```

Comment:
Can you explain why you use `val > 0.42`?  I tested the changes from your PR and I had to change this to `0.43`. With `0.42` it detected every screen as max number of games played and just stopped.

Reply:
when I used it in my small android device, even after capturing the text in the screenshot, the bot unable to detect it and loops clicking the battle button

Reply:
OK. I have refactored quite a bit in the meantime. I had the same problem and added a priority list if multiple images match. See #4 and https://github.com/CodeWithMa/PVPokeLossBot/blob/683f475c3d35d16aac57c4a6a4c37e5082a3d678/src/image_decision_maker.py#L47.

I have cherry picked your other changes for windows support in #3 and merged them.